qt/python/mantidqt/mantidqt/widgets/helpwindow/helpwindowmodel.py

```diff
@@ -0,0 +1,78 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import os
+from qtpy.QtCore import QUrl
+from qtpy.QtWebEngineCore import QWebEngineUrlRequestInterceptor, QWebEngineUrlRequestInfo
+
+
+class NoOpRequestInterceptor(QWebEngineUrlRequestInterceptor):
+    """
+    A no-op interceptor that does nothing. Used if we're not loading local docs.
+    """
+
+    def interceptRequest(self, info: QWebEngineUrlRequestInfo):
```

Comment:
You can probably leave this method off and just have the class be an empty child